api-server/src/server/index.js

```diff
@@ -82,6 +83,14 @@ app.start = _.once(function () {
   });
 });
 
+if (process.env.NODE_ENV === 'development') {
```

Comment:
```suggestion
if (process.env.FREECODECAMP_NODE_ENV === 'development') {
```
We use this one to differentiate between production and development. NODE_ENV is used that way by other software (like Jest and Express), but this is ours.